src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.service.accord;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import accord.impl.SimpleProgressLog;
+import accord.local.Node;
+import accord.messages.Reply;
+import accord.messages.Request;
+import org.apache.cassandra.concurrent.Shutdownable;
+import org.apache.cassandra.net.IVerbHandler;
+import org.apache.cassandra.service.accord.api.AccordAgent;
+import org.apache.cassandra.service.accord.api.AccordScheduler;
+import org.apache.cassandra.utils.Clock;
+import org.apache.cassandra.utils.ExecutorUtils;
+import org.apache.cassandra.utils.FBUtilities;
+
+public class AccordService implements Shutdownable
+{
+    public final Node node;
+    private final Shutdownable nodeShutdown;
+    private final AccordMessageSink messageSink;
+    public final AccordConfigurationService configService;
+    private final AccordScheduler scheduler;
+    private final AccordVerbHandler verbHandler;
+
+    private static class Handle
+    {
+        public static final AccordService instance = new AccordService();
+    }
+
+    public static AccordService instance()
+    {
+        return Handle.instance;
+    }
+
+    public static long uniqueNow()
+    {
+        return TimeUnit.MILLISECONDS.toMicros(Clock.Global.currentTimeMillis());
```

Comment:
nit: time doesn't need to move forward and can go backwards, so this isn't `unique`; this is `java.lang.System#currentTimeMillis` in normal operations

Reply:
Good spot. I'll address this.